modules/EmailMan/EmailMan.php

```diff
@@ -1054,7 +1052,7 @@ public function addOptInEmailToEmailQueue($module, $uid)
 
         if ($foundBean !== null) {
             $emailAddress->retrieve_by_string_fields(array('email_address' => $foundBean->email1));
-            if ($emailAddress->confirm_opt_in === 'opt-in') {
+            if ($emailAddress->getConfirmedOptIn() === SugarEmailAddress::COI_STAT_OPT_IN) {
```

Comment:
I noticed that in some places the SugarEmailAddress and in others it is EmailAddress. It seems a little inconsistent.

Reply:
class EmailAddress extends the ShugarEmailAddress, did you forget it?

Reply:
I didn't forget it. However you have refactored the code base so that the constants are in the SugarEmailAddress. Since you have done that, you also need to update the references to this in the codebase. 

Don't worry about It, will push up the fixes to these issues.